partner_affiliate/tests/test_partner_affiliate.py

```diff
@@ -7,83 +7,89 @@ def setUp(self):
+        self.assertTrue(my_affiliate, "The new affiliate have been created.")
 
-        self.assertEquals(new_affiliate.type, 'affiliate',
+        self.assertEquals(my_affiliate.type, 'affiliate',
                           "Check type must be 'affiliate'")
-        self.assertEquals(new_affiliate.parent_id.id, self.first_parent.id,
+        self.assertEquals(my_affiliate.parent_id.id, self.first_company.id,
                           "Must be child of the parent defined in the setup")
-        self.assertEquals(new_affiliate.street, "affiliate street",
+        self.assertEquals(my_affiliate.street, "affiliate street",
                           "The street have been correctly set.")
-        self.assertEquals(self.first_parent.street, "first parent street",
+        self.assertEquals(self.first_company.street, "first company street",
                           "The parent continues with his original street")
 
         # Change the parent of the affiliate for the second one in the set-up.
-        new_affiliate.parent_id = self.second_parent.id
-        new_affiliate.onchange_parent_id()
+        my_affiliate.parent_id = self.second_company.id
+        my_affiliate.onchange_parent_id()
 
         # The parent have been changed. And is not the first one.
-        self.assertEquals(new_affiliate.parent_id.id, self.second_parent.id)
+        self.assertEquals(my_affiliate.parent_id.id, self.second_company.id)
 
-        # The affiliate keeps its data for the street. Not modified.
-        self.assertEquals(new_affiliate.street, "affiliate street",
+        # The affiliate keeps its data for the street (address). Not modified.
+        self.assertEquals(my_affiliate.street, "affiliate street",
                           "keeps the same street")
         # The data for the street of the first parent have not been changed.
-        self.assertEquals(self.first_parent.street, "first parent street",
+        self.assertEquals(self.first_company.street, "first company street",
                           "keeps the same street")
 
-
     # Check that the default value for 'type' defined by default in the view
     # is set correctly when a new affiliate is created.
     def test_new_affiliate_is_created_with_type_affiliate_by_default(self):
         new_affiliate = self.partner_obj.with_context(
-            {'default_parent_id': self.first_parent.id,
+            {'default_parent_id': self.first_company.id,
              'default_is_company': True,
              'default_type': 'affiliate'
              }
         ).create({
             'name': 'MyTestAffiliate',
             'street': 'affiliate street',
-            'street2': 'number 11',
-            'zip': 567,
-            'city': 'Test City',
-            'email': 'myAffiliate@test.com',
         })
 
         self.assertEquals(new_affiliate.type, 'affiliate')
+
+    # Check that when changing the parent from an individual, it changes also
```

Comment:
Good contribution this test, to check that when we have a contact, it continues working as in the core.

Reply:
thx